src/main/java/bookstore/validation/PasswordValidator.java

```diff
@@ -0,0 +1,14 @@
+package bookstore.validation;
+
+import bookstore.dto.user.UserRegistrationRequestDto;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class PasswordValidator implements ConstraintValidator<FieldMatch, Object> {
+
+    @Override
+    public boolean isValid(Object obj, ConstraintValidatorContext constraintValidatorContext) {
+        UserRegistrationRequestDto user = (UserRegistrationRequestDto) obj;
```

Comment:
Think about avoiding casting. Maybe we can make `obj` argument `UserRegistrationRequestDto ` type?